client/lib/cgutil/cpuset_manager_v2.go

```diff
@@ -54,16 +55,21 @@ type cpusetManagerV2 struct {
 }
 
 func NewCpusetManagerV2(parent string, reservable []uint16, logger hclog.Logger) CpusetManager {
+	if err := minimumRootControllers(); err != nil {
+		logger.Error("failed to enabled minimum set of cgroup controllers; disable cpuset management", "error", err)
+		return new(NoopCpusetManager)
+	}
+
 	parentAbs := filepath.Join(CgroupRoot, parent)
 	if err := os.MkdirAll(parentAbs, 0o755); err != nil {
-		logger.Warn("failed to ensure nomad parent cgroup exists; disable cpuset management", "error", err)
+		logger.Error("failed to ensure nomad parent cgroup exists; disable cpuset management", "error", err)
```

Comment:
Can we change these log entries to say "disabling cpuset management"? "disable" sounds like we're telling the cluster administrator to do it.